motor_controller:
  ros__parameters:
    wheel_base: 0.165
    wheel_diameter: 0.06475
    encoder_ticks_per_rev: 576
    max_speed: 0.5
    max_angular_speed: 2.0
    min_duty_cycle: 80
    max_duty_cycle: 100
    pwm_frequency: 1000
    speed_kp: 100.0
    speed_ki: 10.0
    speed_kd: 0.0
    odom_frame: "odom"
    base_frame: "base_footprint"